src/rdo.rs

```diff
@@ -439,9 +439,14 @@ pub fn compute_distortion_scale<T: Pixel>(
   let x = frame_bo.0.x >> IMPORTANCE_BLOCK_TO_BLOCK_SHIFT;
   let y = frame_bo.0.y >> IMPORTANCE_BLOCK_TO_BLOCK_SHIFT;
 
-  let propagate_cost = fi.block_importances[y * fi.w_in_imp_b + x] as f64;
-  let intra_cost = fi.lookahead_intra_costs[y * fi.w_in_imp_b + x] as f64;
+  //  let propagate_cost = fi.block_importances[y * fi.w_in_imp_b + x] as f64;
+  //  let intra_cost = fi.lookahead_intra_costs[y * fi.w_in_imp_b + x] as f64;
 
+  //  distortion_scale_for(propagate_cost, intra_cost)
+  fi.distortion_scales[y * fi.w_in_imp_b + x]
+}
+
+pub fn distortion_scale_for(propagate_cost: f64, intra_cost: f64) -> f64 {
```

Comment:
This method is only used in api/internal.rs, so it should be moved there (later we could also move all the block importance related code in its own file)

Reply:
I'd do that once we move the block importance code.